kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -43,6 +43,7 @@ def test_config_env_vars(self):
         os.environ['KG_KEYFILE'] = '/test/fake.key'
         os.environ['KG_CERTFILE'] = '/test/fake.crt'
         os.environ['KG_CLIENT_CA'] = '/test/fake_ca.crt'
+        os.environ['KG_SSL_VERSION'] = '5'
```

Comment:
Since the new default (`PROTOCOL_TLSv1_2`) has a value of 5 anyway, I'm not sure this test is ensuring the code is doing the right thing.  I would recommend using '3' and ensuring that.